packages/fastify/modules/index.js

```diff
@@ -0,0 +1,25 @@
+const { version } = require('../package.json');
+const {
+  plugin,
```

Comment:
Renamed this to be inline with the other plugins/middlewares. The idea is that when somebody imports it should be `const { plugin: promsterPlugin } = require('@promster/fastify')`.